daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -397,6 +397,32 @@ def _close(self):
         self._battery_manager.close()
 
 
+class RazerBlackWidowV3ProWireless(RazerBlackWidowV3ProWired):
+    """
+    Class for the Razer BlackWidow v3 Pro 2.4 Ghz Wireless
+    """
+    USB_PID = 0x025C
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_charge_effect', 'set_charge_colour', 'get_battery', 'is_charging', 'set_low_battery_threshold']
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._battery_manager = _BatteryManager(self, self._device_number, "Razer BlackWidow V3 Pro (Wireless)")
```

Comment:
How about adjusting the base class instead to be `self._battery_manager = _BatteryManager(self, self._device_number, "Razer BlackWidow V3 Pro")`, then you don't need to overwrite `self._battery_manager`

Reply:
I have adjusted the the `battery_manager` of the wireless variant to use the one of the wired version.